ios/FluentUI/Extensions/UIColor+Extensions.swift

```diff
@@ -5,35 +5,44 @@
 
 import UIKit
 
-@objc public extension UIColor {
-    private struct AssociatedKeys {
-        static var light: String = "light"
-        static var lightHighContrast: String = "lightHighContrast"
-    }
+public extension UIColor {
```

Comment:
Is it possible to make the extension internal to further reduce binary size?
What would be the use case for the client apps to use/need it?

Reply:
The main reason to make this public is that custom theme definitions may want to use it, especially the multi-color initializer. But I'll make it internal if we end up not exposing UIColor as the type for themes and just ask for a struct filled with integers instead.